README.md

```diff
@@ -1,13 +1,25 @@
 
-## notes for whoever writes this:
+Even though SQL has just one purpose, it is used by many different database systems such as MySQL, PostgreSQL, or the system we'll being using in this course: SQLite.
 
-For the second objective, use the text from [this readme](https://github.com/learn-co-curriculum/why-sql-matters). 
+## Web Developers <3 Databases
 
-For the third objective, outline the basic flow/objectives of the unit. Something like: "by the end of this unit you will know how to create SQLite3 databases, create database tables, insert data into, update the data within, select and delete data from those database tables. You will learn how data within a given database can be related and how to write Ruby programs that talk to your databases and actually save data into them."
+Why do we need to master SQL? As a full-stack web developer, you'll frequently be working with databases to manage the data associated with your applications. Think about the web apps you're familiar with. Facebook saves user data and stores––or persists––your associations to your friends. Amazon has an enormous database of items for sale. The New York Times has a storage system for all of their articles. Most of the sites you interact with every day, and consequently most of the sites you'll work on or build, need databases to persist data.
+
+We've already seen how we can build Ruby programs that model real-world objects and environments. With database management skills, we'll learn to store representations of the Ruby objects our programs create and retrieve them at the appropriate time. We’ll also learn how to connect our Ruby (and even later, Ruby on Rails) applications to our databases. 
+
+For example, a basic web application might have many users. So far, we've learned how to build a Ruby `User` class that produces user objects. But, we don't yet know how to store those users and their details. If a user signs up for our app and we proceed to lose all their information immediately, how will we know if an existing user is signing back into our app? We need a way to take our Ruby objects, store them in a database and retrieve them at the appropriate time.
+
+## In This Unit
+
+This topic will cover how to use and navigate databases based on SQL. By the end of this topic, you’ll be able to: 
+
+* Create SQLite3 databases.
+* Create, update, select, and delete data from database tables.
+* Relate data within a given database.
+* Write SQL code in both your command line and your text editor and execute the code against a database. 
+* Write Ruby programs that talk to and save data to your databases.
```

Comment:
This is really good. One suggestion is to mention that we use SQLite3 in that first paragraph as well as list a couple of databases. This is mentioned in the paragraph that was taken out in the [Intro and Installation Readme](https://github.com/learn-co-curriculum/sql-intro-and-installation/commit/db7dbc69f9e6513c292f0bb657a76bea98fb377f). 

Also, this is a section in the [Database Basics Readme](https://github.com/learn-co-curriculum/database-basics) that I think can be shortened and included in this Readme somewhere. Maybe also after that first paragraph so we can explain why we choose SQLite3

![screen shot 2015-10-21 at 10 54 23 am](https://cloud.githubusercontent.com/assets/1892150/10640431/9e584afa-77e2-11e5-9bdd-40f7c4ece973.png)